nixos/modules/security/pam.nix

```diff
@@ -281,7 +281,7 @@ let
                 "auth optional ${pkgs.pam_mount}/lib/security/pam_mount.so"}
               ${optionalString cfg.enableKwallet
                 ("auth optional ${pkgs.plasma5.kwallet-pam}/lib/security/pam_kwallet5.so" +
-                 " kwalletd=${pkgs.libsForQt5.kwallet}/bin/kwalletd5")}
+                 " kwalletd=${pkgs.libsForQt5.kwallet.bin}/bin/kwalletd5")}
```

Comment:
Can I suggest to use `lib.getBin pkgs.libsForQt5.kwallet` instead? That would ensure that the correct path is found even if the outputs of `kwallet` change later.